### DISCUSSION TITLE
Navigation: Ability to select from favourite subfolders

### DISCUSSION THREAD

**Discussion by andrew6767** [osmandapp/OsmAnd#17006] [Ideas]
My idea in user story format:

_As someone trying to navigate a long route I would like the ability when in the navigation screen to browse to a favourites folder when selecting intermediary stops so that I don't have to scroll through the full list of all my favourited places._

The way that issue OsmAnd is add favourite markers for the route I will drive into their own folders but when you go to the navigation screen it only gives you the option to select from a left > right scrolling list of all your favourite markers which kind of makes the folder system a bit pointless for me.

Hope this is clear but if not happy to try and explain in more detail or get some screenshots.

**Comment by ThomasMeister123** [comment]
You can switch to top-down-scrolling when you click on the left side of left-right-scrolling.

There you have different sort-options

**Reply by andrew6767** [reply]
Amazing! this does what I want. Can't believe I haven't clicked on that left side before.

**Reply by ThomasMeister123** [reply]
Youre welcome. Sometimes its not easy to find a OSMAnd-function. Even if you already used a function and you search it after a long time 🙈
